psi4/src/psi4/libfock/PK_workers.cc

```diff
@@ -40,10 +39,10 @@ namespace psi {
 
 namespace pk {
 
-AOShellSieveIterator::AOShellSieveIterator(std::shared_ptr<BasisSet> prim, SharedSieve sieve_input)
-    : shell_pairs_(sieve_input->shell_pairs()) {
+AOShellSieveIterator::AOShellSieveIterator(std::shared_ptr<BasisSet> prim, SharedSieve eri_input)
```

Comment:
Is `SharedSieve` accurate, should it be named e.g. `ERIEvaluator`?

Reply:
Ah, nice catch! Given it is a `shared_ptr` to a `TwoBodyAOInt` object, something like `SharedERI` or `SharedInt` would probably be the best name for it.